xbmc/filesystem/AudioBookFileDirectory.h

```diff
@@ -8,10 +8,13 @@
 #pragma once
 
 #include "IFileDirectory.h"
+#include "music/tags/MusicInfoTag.h"
```

Comment:
unrelated to the PR but for some reason that header is not included in cmake?

Reply:
Dunno ???

Reply:
> for some reason that header is not included in cmake?

Should definitely be added to xbmc/filesystem/CMakeLists.txt . Seems this was forgotten when the file was introduced.